eng/common/scripts/get-codeowners.ps1

```diff
@@ -1,49 +1,50 @@
 param (
-  $TargetDirectory, # should be in relative form from root of repo. EG: sdk/servicebus
-  $RootDirectory, # ideally $(Build.SourcesDirectory)
-  $VsoVariable = "" # target devops output variable
+  [string]$CodeOwnerPathExpression, # Code path to code owners. e.g sdk/core/azure-amqp
+  [string]$ToolVersion = "", # Placeholder. Will update in next PR
+  [string]$ToolPath = "$env:AGENT_TOOLSDIRECTORY", # The place to check the tool existence. Put $(Agent.ToolsDirectory) as default
```

Comment:
We might want to actually consider a temp directory for this. That is what @danieljurek is doing for cspell, that would allow for easier running locally as well. We can always change it in DevOps if we want.

Reply:
I've been using [this](https://github.com/Azure/azure-sdk-tools/blob/55b4b3f232508b3e4620cddeddbf1d761e355516/eng/common/TestResources/New-TestResources.ps1#L144) for cross-platform temp directory lookup.

Reply:
I've been using something similar: 

```pwsh
[Parameter()]
[string] $WorkingDirectory = (Join-Path ([System.IO.Path]::GetTempPath()) ([System.IO.Path]::GetRandomFileName())),
```

This generates a temporary folder name which can then be created later if it doesn't already exist.